usaspending_api/conftest_helpers.py

```diff
@@ -123,20 +144,7 @@ def _add_contents(self, **options):
 
     @classmethod
     def _generate_index_name(cls):
-        return "test-{}-{}".format(
-            datetime.now(timezone.utc).strftime("%Y-%m-%d-%H-%M-%S-%f"), generate_random_string()
-        )
-
-    @staticmethod
-    def convert_json_arrays_to_list(json_array: Optional[List[dict]]) -> Optional[List[str]]:
```

Comment:
This is used above on line 112 to build out the ES record. From what I remember, our test suite for ES does not leverage the ES ETL and ES Indexer, but instead inserts each record directly into Elasticsearch Index.

Reply:
I removed the line referenced in the follow-up branch which is fixing tests and any code issues (using another branch in an attempt to help PR reviews)